### ISSUE TITLE
tsc.exe exited with code 1

### ISSUE THREAD

**Issue by Matiszak** [aspnet/Tooling#760]
Hi,
I'm making my project in VS2015 with angular2 in typescript. Occasionaly i've encountered error:
error MSB6001: Invalid command line switch for "tsc.exe". Illegal characters in path. 
I don't remember how i got over it last time, i supposed it disappeared without my interaction. But today this error doesn't want to go away.
I installed TS 2.0 RC because i thought it was a bug and it had been fixed in new version. But now i get error
Severity	Code	Description	Project	File	Line	Suppression State
Error	MSB6006	"tsc.exe" exited with code 1.	MGMS.Web	C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\TypeScript\Microsoft.TypeScript.targets	222	

Can you help me with this ?

Here is my diagnostic log
[diagnostic log.txt](https://github.com/aspnet/Tooling/files/453265/diagnostic.log.txt)

and project(external file uploader because github didn't accept my zip :( )
http://www.filedropper.com/project_11

**Comment by Matiszak** [comment]
I tried to figure it out and changed the Microsoft.TypeScript.targets file to use hardcoded value 1.8 of the tools. It worked. I don't know how because before installation of Typescript 2.0 it didn't work as well but it worked. To be sure it works for all my team member i added two lines in xproj 

  <PropertyGroup>
    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0</VisualStudioVersion>
    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
    **<TypeScriptNuGetToolsVersion>1.8</TypeScriptNuGetToolsVersion>
    <TypeScriptToolsVersion>1.8</TypeScriptToolsVersion>**
  </PropertyGroup>

I also tried to run continous integration on my solution in visualstudioonline. But here it doesn't work with the same error 1 of tsc.exe. It's funny because the log shows me that visualstudioonline correctly uses tsc.exe in version 1.8. What is going on guys ?

Attached log from agent
[1_Build.txt](https://github.com/aspnet/Tooling/files/453929/1_Build.txt)

I tried running tsc,exe v2.0 from command line and it shows like my log in first post 
![image](https://cloud.githubusercontent.com/assets/20770017/18230961/1dacb534-72ab-11e6-8e84-692ef57efcb9.png)
